component/sections/portfolio/index.js

```diff
@@ -11,13 +11,14 @@ const Variants = {
 function Portfolio({ template, data }) {
   const variant = data?.variant || data?.variants?.condition;
   const Variant = Variants?.[variant];
-
+  console.log(data?.variants);
   const props = {
     template,
     caption: data?.variants?.subtitle,
     title: data?.variants?.title,
     portfolio: data?.variants?.portfolio,
-    portfoliosWithCategory: data?.variants?.portfolios,
+    portfoliosWithCategory: data?.variants?.portfoliosWithCategories,
+    portfolios: data?.variants?.portfolios,
```

Comment:
It seems that the `portfolios` prop duplicates `portfolio` ?
Based on the _**portfolios**_ function from [sanity-packages fields
](https://github.com/webriq-pagebuilder/sanity-packages/blob/reusable_sections/packages/sanity-plugin-schema-default/src/schemas/common/fields.js) the field name is `portfolio`. This is the same function imported on the [portfolios schema](https://github.com/webriq-pagebuilder/sanity-packages/blob/reusable_sections/packages/sanity-plugin-schema-default/src/schemas/sections/portfolio/schema/index.js).

Reply:
I am using portfolios name on schema under studio.